.gitlab-ci.yml

```diff
@@ -60,6 +60,11 @@ deploy-gcp-staging:
     #- docker login -u "gitlab-ci-token" -p "$CI_BUILD_TOKEN" $CI_REGISTRY
     - export HELM_REPLICA_COUNT=$HELM_REPLICA_COUNT_STAGING
     - export SECRET_KEY_BASE=$SECRET_KEY_BASE_STAGING
+    - export GCP_DATABASE_HOST=$GCP_DATABASE_HOST_STAGING
```

Comment:
heroku and k8s CI jobs use different VMs so you can use the same environment variable name, they'll not overlap each other. And make this rule applied for travis-ci deployment (try to use https://docs.travis-ci.com/user/build-stages/)